torchinductor/decomposition.py

```diff
@@ -32,6 +32,8 @@
         aten.embedding_dense_backward,
         aten.expand_as,
         aten.flip,
+        aten.floor_divide.default,
+        aten.fmod.Tensor,
```

Comment:
even though nominally we have fmod decomposition, attempting to use it leads to 
```
def forward(self, arg0_1, arg1_1):
    fmod = torch.ops.prims.fmod.default(arg0_1, arg1_1);  arg0_1 = arg1_1 = None
    return (fmod,)
    
torchinductor.graph: [WARNING] Creating implicit fallback for:
  target: prims.fmod.default
  args[0]: TensorBox(StorageBox(
    InputBuffer(name='arg0_1', layout=FixedLayout('cuda', torch.int32, size=[s0], stride=[1]))
  ))
  args[1]: TensorBox(StorageBox(
    InputBuffer(name='arg1_1', layout=FixedLayout('cuda', torch.int32, size=[s0], stride=[1]))
  ))
torchinductor.ir: [WARNING] Using FallbackKernel: torch.ops.prims.fmod.default

```
(because decomposition just calls directly into prim)
so again we'd either have to write a real decomposition on the pytorch side, or lowering on the inductor side to support fmod.